webpack.config.js

```diff
@@ -0,0 +1,35 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin'); // eslint-disable-line
+
+module.exports = {
+  entry: {
```

Comment:
- [ ] [OPTIONAL] It's be a great idea to add an overflow to auto to your modal on desktop in order to make the page fully visible to the user.

<img width="945" alt="image" src="https://user-images.githubusercontent.com/52934128/218267072-1a94a88c-a93f-4c9f-8fb0-77358b25ef37.png">